Doc/library/stdtypes.rst

```diff
@@ -1653,6 +1653,14 @@ expression support in the :mod:`re` module).
    from the "Alphabetic" property defined in the Unicode Standard.
 
 
+.. method:: str.isascii()
+
+   Return true if all characters in the string are ASCII, false otherwise.
+   ASCII characters are characters which :func:`ord` returns less than 128.
```

Comment:
Just a suggestion: We way reuse `curses.ascii.isascii()` documentation to describe what we meant by an ASCII character: https://docs.python.org/3/library/curses.ascii.html#curses.ascii.isascii

Reply:
I suggest: "ASCII characters have code points in the range U+0000-U+007F."